speculos/mcu/ocr.py

```diff
@@ -150,63 +118,190 @@ def _get_font_map(font: bagl_font.Font) -> Mapping[Char, BitMapChar]:
     return font_map
 
 
-def find_char_from_bitmap(bitmap: BitMap):
-    """
-     Find a character from a bitmap
-    >>> font = get_font(4)
-    >>> char = get_char(font, 'c')
-    >>> find_char_from_bitmap(char.bitmap)
-    'c'
-    """
-    all_values = []
+class OCR:
```

Comment:
There is no _class_ property which is accessed via `self.<property>`, in OCR class.
I used class property to avoid loading/storing font data several times, but if only one instance of OCR exist, there is no reason to use class properties => I will use instance properties instead.